homeassistant/components/jvc_projector/binary_sensor.py

```diff
@@ -20,7 +19,7 @@ async def async_setup_entry(
     hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
     """Set up the JVC Projector platform from a config entry."""
-    coordinator: JvcProjectorDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    coordinator: JvcProjectorDataUpdateCoordinator = entry.runtime_data
```

Comment:
```suggestion
    coordinator = entry.runtime_data
```
This way we can omit this

Reply:
good for me